lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -2460,7 +2542,7 @@ class EditableTextGeometry {
   void applyToDomElement(DomHTMLElement domElement) {
     final String cssTransform = float64ListToCssTransform(globalTransform);
     domElement.style
-      ..width = '${width}px'
+      ..width = '${width - 2}px'
```

Comment:
Our inputs don't take into account the scrollbar width, but the framework inputs do.   So we can sometimes end up running into an edge case where the framework text creates a line-break at the very end of a multiline input, but our inputs don't.  Slightly adjusting the width of our input to account for the scrollbar fixes this - but I'm wondering if there's a more reliable way to solve this than what I'm doing here. 

Would also need to cross-test this with different platform and browser combinations